orchestrator/gorc/src/commands/keys/cosmos.rs

```diff
@@ -0,0 +1,37 @@
+//! `cosmos keys` subcommand
+
+mod add;
+mod delete;
+mod import;
+mod list;
+mod show;
+mod update;
+
+use abscissa_core::{Command, Options, Runnable};
+
+#[derive(Command, Debug, Options, Runnable)]
+pub enum CosmosKeysCmd {
+    #[options(help = "add [name]")]
+    Add(add::AddCosmosKeyCmd),
+
+    #[options(help = "import [name] [mnemnoic]")]
```

Comment:
Should we prompt the mnemonic instead of adding it as an argument so that the mnemonic is not in history?

Reply:
This is nice, but for scripting (i.e. test environments) we still need ability to do it non-interactively